numba/compiler.py

```diff
@@ -460,21 +462,31 @@ def _backend(self, lowerfn, objectmode):
         if self.library is None:
             codegen = self.targetctx.jit_codegen()
             self.library = codegen.create_library(self.bc.func_qualname)
-        lowered = lowerfn()
+        if config.DISABLE_JIT and not self.flags.force_enable_jit:
```

Comment:
I would hope not :) It's not a documented API. There are a few results on Google but that looks quite marginal...